helper/cssimport.php

```diff
@@ -421,7 +421,6 @@ protected function explodeBorderShorthand (&$decls) {
             $border_color_set = false;
             while ( $index < 3 ) {
                 if ( $border_width_set === false ) {
-                    if (!isset($values [$index])) $values [$index] = 'medium';
```

Comment:
I did not look in detail to cssimport.php, but these fallback values where not needed?

Reply:
Hello @Klap-in, thank you for your feedback! I had to download ODT plugin 2019-07-27 release and backport all commits to find those that breaks the ODT export functionality. This line you cited was added by me as a first attempt to fix the PHP 8 Warning error, but I find the ternite user solution better, which he/she set a null value if it isn't set, to keep PHP 7 behavior. I removed the line but I forgot to make the null verification for some reason (I took about 6 hours to port and test the ODT plugin code to make sure it's working). I make a new commit to make this verification. Thank you!